### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by Laulen** [HZAI-ZJNU/Mamba-YOLO#43]
Has anyone encountered this problem? The first round of training revealed the following errors.
<img width="887" alt="image" src="https://github.com/user-attachments/assets/f5b268f7-5494-4717-b903-0e3a003e7ad5">

**Comment by yazLL** [comment]
I have exactly the same error, have you resolved it?
Ubuntu 20.04 RTX3090
conda pytorch 2.3
cuda 11.8

python mbyolo_train.py --device 0 --batch 32 --epochs 100 --task train --data ultralytics/cfg/datasets/coco.yaml --config ultralytics/cfg/models/mamba-yolo/Mamba-YOLO-T.yaml --amp  --project ./output_dir/mscoco --name mambayolo_n

changed sorce data to my path in coco.yaml
path: /home/ml/250GB/Mamba-YOLO-main/datasets/MSCOCO2017 # dataset root dir
others not been touched

**Comment by zhang123-lf** [comment]
> 我有完全一样的错误，你解决了吗？ Ubuntu 20.04 RTX3090 conda pytorch 2.3 cuda 11.8
> 
> python mbyolo_train.py --device 0 --batch 32 --epochs 100 --task train --data ultralytics/cfg/datasets/coco.yaml --config ultralytics/cfg/models/mamba-yolo/Mamba-YOLO-T.yaml --amp --project ./output_dir/mscoco --name mambayolo_n
> 
> 将源数据更改为 coco.yaml 路径中的我的路径：/home/ml/250GB/Mamba-YOLO-main/datasets/MSCOCO2017 # 数据集根目录 其他目录未受影响
> 
> [已解决] 错误的文件夹结构导致无法找到标签，从而导致验证错误

请教一下，在训练完成后验证集上的AP等参数为0，这可能是什么原因呢，是需要在源代码是做什么修改吗

**Comment by yws-wxs** [comment]
> Has anyone encountered this problem? The first round of training revealed the following errors. <img alt="image" width="887" src="https://private-user-images.githubusercontent.com/53033753/366049121-f5b268f7-5494-4717-b903-0e3a003e7ad5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ0MDcxMDUsIm5iZiI6MTc0NDQwNjgwNSwicGF0aCI6Ii81MzAzMzc1My8zNjYwNDkxMjEtZjViMjY4ZjctNTQ5NC00NzE3LWI5MDMtMGUzYTAwM2U3YWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDIxMjY0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzNjU4MmIxMDQwMjQzNWZlMmEyZWRlOWQwM2YwMmJjNzBmNDk3ZDUxMzE1Mjg2YjFkNDBhOTJjNDU3NzJhNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.TmGATBLt3e_i2QYIBxYAxMxpbCnvC_D61faMBj1dhYU">

The coco dataset labels are not done properly
